tests/Zend/Ldap/OriginalOfflineTest.php

```diff
@@ -60,7 +60,7 @@ public function setUp()
     public function testFilterEscapeBasicOperation()
     {
         $input = 'a*b(b)d\e/f';
-        $expected = 'a\2ab\28b\29d\5ce\2ff';
+        $expected = 'a\2ab\28b\29d\5ce/f';
```

Comment:
is this change necesary? `\2f` should be a forward-slash.

Reply:
@falkenhawk yes this one is necessary, they changed it the same way in laminas. That's just the expected result now.